proxies/live/apiproxy/policies/JavaScript.ValidateClaimFromExternalIdToken.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<Javascript async="false" continueOnError="false" enabled="true" timeLimit="200" name="JavaScript.ValidateClaimFromExternalIdToken">
+    <DisplayName>JavaScript.ValidateClaimFromExternalIdToken</DisplayName>
+    <Properties/>
```

Comment:
should properties be left empty like this? Is it mandatory or can simply by removed?

Reply:
code from immunisation api